src/Api/Orbit.cpp

```diff
@@ -87,15 +87,33 @@ void orbit_api_async_string(const char* str, uint64_t id, orbit_api_color color)
   EnqueueApiEvent<orbit_api::ApiStringEvent>(str, id, color);
 }
 
-static void orbit_api_initialize_v0(orbit_api_v0* api) {
-  // The api function table is accessed by user code using this pattern:
-  //
-  // bool initialized = api.initialized;
-  // std::atomic_thread_fence(std::memory_order_acquire)
-  // if (initialized && api->enabled && api->orbit_api_function_name) api->orbit_api_function_name()
-  //
-  // We use acquire and release semantics to make sure that when api->initialized is set, all the
-  // function pointers have been assigned and are visible to other cores.
+template <typename orbit_api_current>
```

Comment:
nit. our templated types usually end with "T". Also, since this is c++, we should probably use camel case, OrbitApiT?

Reply:
Done.